guidelines/sc/21/label-in-name.html

```diff
@@ -1,11 +1,12 @@
 <section class="sc new">
-   <h4>Accessible Name</h4>
+   <h4>Label in Name</h4>
    <p class="conformance-level">A</p>
    <p class="change">New</p>
 	<p class="ednote">Discussion of the issue is available in <a href="https://github.com/w3c/wcag21/issues/68">Issue 68</a>. To file comments on this proposal, please <a href="https://github.com/w3c/wcag21/issues/new">raise new issues</a> for each discrete comment in GitHub.
 	</p>
 	
-  <p>Where an active control has a visible label, the <a>accessible name</a> for the control includes the text string used for its visible label.</p>
-  <p class="note">A best practice is to place the text string at the start of the text string.</p>
-
+  <p>For active <a>user interface components</a> with <a>labels</a> that include text, the <a>name</a> includes the text of the label.</p>
```

Comment:
For active user interface components with labels that include text or images of text, the name includes the text of the label.

Reply:
Hmmm.... good catch, David.  I agree with this change.  The original wording is actually very unclear here because it refers to the "text string" of the "visible label", so it sort of eliminates an image of text as a possible label.  My version catches the fact that a label is always visible per the definition, and also that a label could have no visible text (e.g. an icon)... but I missed the possibility of an image of text.

@awkawk, do you think we should fix this here or put in a new issue?  I know you also wanted to remove "active" as well anyway (which I think I agree with).